staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/crd-schemas.json

```diff
@@ -0,0 +1,252 @@
+{
+  "definitions": {
+    "io.k8s.example.NoSchema": {
+      "type": "object",
+      "x-kubernetes-group-version-kind": [
+        {
+          "group": "example.k8s.io",
+          "kind": "NoSchema",
+          "version": "v1"
+        }
+      ]
+    },
+    "io.k8s.example.LargeStatus": {
+      "type": "object",
+      "x-kubernetes-group-version-kind": [
+        {
+          "group": "example.k8s.io",
+          "kind": "LargeStatus",
+          "version": "v1"
+        }
+      ],
+      "properties": {
+        "apiVersion": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta"
+        },
+        "spec": {
+          "type": "object"
+        },
+        "status": {
+          "type": "object",
+          "properties": {
+            "items": {
+              "type": "object",
+              "x-kubernetes-map-type": "granular",
+              "additionalProperties": {
+                "type": "object",
+                "properties": {
```

Comment:
is this a large status schema?

Reply:
yeah, I don't know if it matters if it is spec of status for this benchmark. The reason I ended up with status here is that CRD I was modeling this off of  (argo workflow) had a large status.

Reply:
I wouldn't be opposed to moving this to the spec.

The update call used here is a low level SSA one that can modify either spec of status.

Reply:
Oh, @liggitt were you asking why the schema isn't that large? I've got this PR mis-phrased, I'm trying to demonstrate a large custom resource, the CRD is pretty small. I'll update a few things to clear that up.

Reply:
yes, that was what I was asking... got it